src/gui/guiFormSpecMenu.cpp

```diff
@@ -3000,7 +3000,8 @@ bool GUIFormSpecMenu::preprocessEvent(const SEvent& event)
 			gui::IGUIElement *focused = Environment->getFocus();
 			if (focused && isMyChild(focused) &&
 					(focused->getType() == gui::EGUIET_LIST_BOX ||
-					 focused->getType() == gui::EGUIET_CHECK_BOX)) {
+					 focused->getType() == gui::EGUIET_CHECK_BOX) &&
```

Comment:
Indentation looks weird, please remove any spaces.

Reply:
It's logical, as it's within the parentheses.

Reply:
I know it's done to line stuff up, but we have a strict rule to never mix spaces and tabs for indentation.